R/ic_dataframe.R

```diff
@@ -17,30 +17,53 @@
 #' identical(x, x_df2)
 ic_dataframe <- function(x) {
 
-  if(methods::is(object = x, class2 = "data.frame")) {
+  if (methods::is(object = x, class2 = "data.frame")) {
```

Comment:
:+1: We could also apply an auto-formatter (maybe that's what you did).

Reply:
No, I do it manually :thinking: What autoformatter are you using?

Reply:
{styler} sometimes

Reply:
And {styler.equals} for equals assignment.